File: JavaScript/8js 2/ilnm.js
/*
|--------------------------------------------------------------------------
| A Strange Trip to the Market
|--------------------------------------------------------------------------
|
*/

function isLockNessMonster(s) {
   return s.includes("tree fiddy") || s.includes("3.50") || s.includes("three fifty");
}


function isLockNessMonster(s) {
   return /tree fiddy|3.50|three fifty/.test(s);
}


function isLockNessMonster(s) {
   const x = ["tree fiddy", "3.50"];
   return x.some((el) => s.includes(el));
}


console.log(
   isLockNessMonster("Your girlscout cookies are ready to ship. Your total comes to tree fiddy"),
   true
);
console.log(
   isLockNessMonster(
      "Yo, I heard you were on the lookout for Nessie. Let me know if you need assistance."
   ),
   false
);

File: Python/8py 2/ilnm.py
"""
|--------------------------------------------------------------------------
| A Strange Trip to the Market
|--------------------------------------------------------------------------
|
"""

def is_lock_ness_monster(string):
   return "tree fiddy" in string or "3.50" in string or "three fifty" in string


from re import search
def is_lock_ness_monster(string):
   return bool(search(r'three fifty|tree fiddy|3\.50', string))


def is_lock_ness_monster(string):
   return any(i in string for i in ('tree fiddy', 'three fifty', '3.50'))


print(is_lock_ness_monster("Your girlscout cookies are ready to ship. Your total comes to tree fiddy"), True)
print(is_lock_ness_monster("Yo, I heard you were on the lookout for Nessie. Let me know if you need assistance."), False)